indexes/private.js

```diff
@@ -20,15 +20,13 @@ module.exports = function (dir, keys) {
   const encryptedFile = path.join(indexesPath(dir), 'encrypted.index')
   const canDecryptFile = path.join(indexesPath(dir), 'canDecrypt.index')
 
-  function save(filename, seq, arr, cb) {
```

Comment:
The `seq` is always the `latestSeq` variable, so I just put that one directly inside here since it has access to that variable anyway.

I removed the cb, because we don't care about the results. If they succeed or fail, we don't care. Except maybe one day we want to put a `debug` for the error case, but that can be done locally, without putting the error in a `cb`